vars/pipelineCapsuleSystemTests.groovy

```diff
@@ -146,6 +146,7 @@ void call() {
                         string(name: 'SYSTEM_TEST_BUILD_NUMBER', value: downstreamBuild.getNumber() as String),
                         string(name: 'SUIT_NAME', value: name),
                         string(name: 'JENKINS_SHARED_LIB_BRANCH', value: params.JENKINS_SHARED_LIB_BRANCH),
+                        string(name: 'NODE_LABEL', value: (params.NODE_NAME ?: ''))
```

Comment:
wouldn't it be better to be consistent and use NODE_LABEL instead of NODE_NAME?

Reply:
of course yess